integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -52,6 +58,23 @@ describe(`[gatsby-source-wordpress] Run tests on develop build`, () => {
       await gatsbyCleanBeforeAll()
     }
 
+    if (isWarmCache && !process.env.WORDPRESS_BASIC_AUTH) {
+      console.log(
+        `Please add the env var WORDPRESS_BASIC_AUTH. It should be a string in the following pattern: base64Encode(\`\${username}:\${password}\`)`
+      )
+
+      await new Promise(resolve => setTimeout(resolve, 100))
+      process.exit(1)
+    }
+
+    if (isWarmCache) {
+      const response = await mutateSchema()
+      console.log(response)
```

Comment:
Do we need these `console.logs`?

Reply:
I left them in because if this fails it'll be hard to spot as the tests will keep running. Since we're doing this outside of tests in `beforeAll` I don't think we can assert anything here